config/tests/jobs/jobs_test.go

```diff
@@ -1115,8 +1121,7 @@ func TestK8sInfraProwBuildJobsCIPolicy(t *testing.T) {
 	jobsToFix := 0
 	jobs := allStaticJobs()
 	for _, job := range jobs {
-		// Only consider Pods destined for the k8s-infra-prow-builds cluster
-		if job.Spec == nil || job.Cluster != "k8s-infra-prow-build" {
+		if job.Spec == nil || (!isCritical(job.Cluster) && !isEKSCluster(job.Cluster)) {
```

Comment:
I think later we should swap this for `isCommunity` or something but right now we seem to be missing the `k8s-infra-prow-builds-trusted` cluster here and that's going to be a big mess, so let's punt.